src/Symfony/Component/Config/Builder/Property.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Builder;
+
+/**
+ * Represent a property when building classes.
+ *
+ * @intenal
```

Comment:
typo: internal (same in Method)

but if we can get rid of the `Property` and `Method` classes altogether, that might be even better :)